game/scripts/npc/abilities/skeleton_king_mortal_strike.txt

```diff
@@ -30,7 +30,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "crit_mult"                                       "300"
+        "crit_mult"                                       "300 315 330 345 360" //OAA
```

Comment:
Revert this. You are changing this just because Pigeon said: '' I would rather have PA with higher crit dmg''. This change is ugly and doesn't change anything. If you buffed crit chance that would be much much better.

Reply:
ok